src/components/Filters/Filters.tsx

```diff
@@ -24,20 +24,22 @@ interface FilterProps<F, G extends Value = string> {
   };
   /** Specifies the layout of the filters. If not supplied it will use the default (horizontal) layout. Using the 'vertical' layout will also remove the "More Filters" button/modal */
   vertical?: boolean;
+  /** Specifies the number of filters before more filters modal  */
+  moreFilters?: number;
```

Comment:
Suggestion: Hm... maybe `numberOfInlineFilters`? 

Ah huh, that's actually pretty close to the `numberOfPageFilters` const down below.

Seems like we could just have the user pass that directly? `numberOfPageFilters` or `numberOfInlineFilters`.

@bdow wdyt?

Reply:
Cool, fwiw @aTrujilloHomeBound if you don't mind `numberOfInlineFilters`, I have a small preference for that one. Thanks!

Reply:
I like it more as well thank you 👍